### PR TITLE
Release 3.7.7

### PR DESCRIPTION
**Pull request by konraddysput**
# Change log

## Version 3.7.7
- Added a new attribute to native reports - `breadcrumbs.lastId`
- Fixed invalid `BacktraceCredentials` behavior when the user passed an invalid Backtrace URL
- Capture invalid native crash reporting handler path in the initialize method

### CODE REVIEW

File: backtrace-library/src/androidTest/java/backtraceio/library/BacktraceClientBreadcrumbsTest.java [hunk 1]
```diff
@@ -1,12 +1,14 @@
 package backtraceio.library;
 
+import static junit.framework.TestCase.assertTrue;
```
Comment (BartoszLitwiniuk) [inline backtrace-library/src/androidTest/java/backtraceio/library/BacktraceClientBreadcrumbsTest.java:3]: unused

File: backtrace-library/src/androidTest/java/backtraceio/library/BacktraceCredentialsTest.java [hunk 2]
```diff
@@ -1,8 +1,11 @@
 package backtraceio.library;
 
 import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertTrue;
```
Comment (BartoszLitwiniuk) [inline backtrace-library/src/androidTest/java/backtraceio/library/BacktraceCredentialsTest.java:5]: unused

**Review by BartoszLitwiniuk — APPROVED** [review]
lgtm

File: backtrace-library/src/androidTest/java/backtraceio/library/BacktraceCredentialsTest.java (hunk 2 above)
Comment (konraddysput) [reply backtrace-library/src/androidTest/java/backtraceio/library/BacktraceCredentialsTest.java:5]: done.

File: backtrace-library/src/androidTest/java/backtraceio/library/BacktraceClientBreadcrumbsTest.java (hunk 1 above)
Comment (konraddysput) [reply backtrace-library/src/androidTest/java/backtraceio/library/BacktraceClientBreadcrumbsTest.java:3]: done

**Review by rick-bt — APPROVED** [review]

**Review by melekr — APPROVED** [review]

**Pull request merged**